ess/src/main/java/org/eclipse/tractusx/ess/notification/mock/MockedNotificationReceiverEndpoint.java

```diff
@@ -0,0 +1,108 @@
+package org.eclipse.tractusx.ess.notification.mock;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.validation.Valid;
+
+import io.swagger.v3.oas.annotations.Hidden;
+import lombok.RequiredArgsConstructor;
+import org.eclipse.tractusx.edc.EdcSubmodelFacade;
+import org.eclipse.tractusx.edc.exceptions.EdcClientException;
+import org.eclipse.tractusx.edc.model.notification.EdcNotification;
+import org.eclipse.tractusx.edc.model.notification.EdcNotificationHeader;
+import org.eclipse.tractusx.ess.discovery.EdcDiscoveryMockConfig;
+import org.eclipse.tractusx.ess.service.SupplyChainImpacted;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+/**
+ * Endpoint for receiving EDC notifications and responding with Mocked BPN results.
+ */
+@RestController
+@RequestMapping("/ess/mock/notification")
+@RequiredArgsConstructor
+@Validated
+@Hidden
+public class MockedNotificationReceiverEndpoint {
+
+    private final EdcDiscoveryMockConfig edcDiscoveryMockConfig;
+    private final EdcSubmodelFacade edcSubmodelFacade;
+
+    @PostMapping("/receive")
+    public void receiveNotification(final @Valid @RequestBody EdcNotification notification) throws EdcClientException {
+        final Optional<String> incidentBpn = Optional.ofNullable(notification.getContent().get("incidentBpn"))
+                                                     .map(Object::toString);
+
+        if (incidentBpn.isPresent()) {
+            final String bpn = incidentBpn.get();
+
+            if (edcDiscoveryMockConfig.getMockEdcResult().containsKey(bpn)) {
+                final SupplyChainImpacted supplyChainImpacted = edcDiscoveryMockConfig.getMockEdcResult().get(bpn);
+
+                final String notificationId = UUID.randomUUID().toString();
+                final String senderEdc = ServletUriComponentsBuilder.fromCurrentRequest().build().toUriString();
+                final String senderBpn = notification.getHeader().getRecipientBpn();
+                final String recipientBpn = notification.getHeader().getSenderBpn();
+                final String recipientUrl = notification.getHeader().getSenderEdc();
+                final Map<String, Object> notificationContent = Map.of("result", supplyChainImpacted.getDescription());
+
+                final EdcNotification edcRequest = edcRequest(notificationId, senderEdc, senderBpn, recipientBpn, notificationContent);
+
+                final var response = edcSubmodelFacade.sendNotification(recipientUrl, "notify-request-asset", edcRequest);
```

Comment:
We need to call the response asset, as that is what we used to register our response receiver endpoint:
```suggestion
                final var response = edcSubmodelFacade.sendNotification(recipientUrl, "ess-response-asset", edcRequest);
```